src/main/webapp/dev/js/common/statusMessage.es6

```diff
@@ -24,6 +24,7 @@ function populateStatusMessageDiv(message, status) {
     $statusMessageDivContent.addClass('alert');
     // Default the status type to info if any invalid status is passed in
     $statusMessageDivContent.addClass(`alert-${StatusType.isValidType(status) ? status : StatusType.INFO}`);
+    $statusMessageDivContent.addClass(`icon-${StatusType.isValidType(status) ? status : StatusType.INFO}`);
```

Comment:
Now that this check will be done twice, might as well extract it into an if statement for more concise and readable code:
```javascript
// Default the status type to info if any invalid status is passed in
if (!StatusType.isValidType(status)) {
    status = StatusType.INFO;
}
$statusMessageDivContent.addClass(`alert-${status} icon-${status}`);
```
You may even want to consider combining all the classes (`overflow-auto`, `alert`, `statusMessage`) into that one line.